rpcserver.go

```diff
@@ -7136,8 +7141,19 @@ func (r *rpcServer) AddInvoice(ctx context.Context,
 
 	// Now that we have the accepted quote, we know the amount in Satoshi
 	// that we need to pay. We can now update the invoice with this amount.
-	mSatPerUnit := acceptedQuote.AskPrice
-	iReq.ValueMsat = int64(req.AssetAmount * mSatPerUnit)
+	askAssetRate := rfqmath.FixedPoint[rfqmath.GoInt[uint64]]{
+		Coefficient: rfqmath.NewGoInt(
```

Comment:
I don't think we want to be using the go int variant anywhere. In this case, you may be silently losing precision.

Reply:
Yes, good catch. I've since tried to use `BigInt` fixed-point everywhere. This got through by mistake.